pegasus/sites.v3/csedweek.org/views/map.haml

```diff
@@ -22,14 +22,18 @@
   var resizeZoom;
   var scrollwheelOption = false;
   var draggableOption = false;
+  var resizeThrottleTimeout;
```

Comment:
i'd mention that this is an Id vs. a timeout value in the variable name.  should also make sure that we're not calling clearTimeout with an undefined value, unless clearTimeout is explicitly compatible with that (i'm not sure).